### PR TITLE
work and about sections of mobile first portfolio project 

### PR DESCRIPTION
**Pull request by Zuhaib042**
I have created these two sections of the portfolio project using my knowledge of Flexbox and Grid layout systems.i have already ran linters and there is no error but still it needs to be reviewed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,18 +11,179 @@
...
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <!-- works section -->
+
+        <section class="works">
+            <h1 class="works-title">My Projects</h1>
```
Comment (mugishasam123) [inline index.html]: - [x] Please Try to use headings properly to improve your page's accessibility and for SEO reasons. so you need to have one `h1` tag on your pages and other headings can be used multiple times on a page. you got this 👍

File: index.html [hunk 2]
```diff
@@ -11,18 +11,179 @@
...
+                            <h6>CANOPY</h6>
+                            <div class="circle"></div>
+                            <p>Back End Dev</p>
+                            <div class="circle"></div>
+                            <p>2015</p>
+                        </div>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </div>
+                    <ul class="tags d-flex list-none">
+                        <li>html</li>
```
Comment (mugishasam123) [inline index.html:92]: - [ ] Please try to stick to figma design as much as possible to make your project looks more professional . you can do so by trying by matching like colors, font-size ,font-weight etc.

Figma design:
image

Your design:
image

**Review by mugishasam123 — CHANGES_REQUESTED** [review]
Hi @Zuhaib042  👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
alt
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors 👍 
- Nice use of git-flow 🔢 
- Good commit messages ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Zuhaib042) [reply index.html:92]: got it. i have fixed it now

File: index.html [hunk 3]
```diff
@@ -11,18 +11,179 @@
 
 <body>
     <header>
-        <nav id="nav">
+        <nav id="nav" class="d-flex">
             <a href="#" class="logo">
                 <p>Bonjour</p>
             </a>
             <div><img src="./images/menu.svg" alt="icon_menu" id="menu"></div>
```
Comment (Qoosim) [inline index.html:18]: - [OPTIONAL] It is not a requirement part of this section of the project, but highly recommended to look into it. The hamburger icon on the top right of your project is broken. Kindly fix it.

**Review by Qoosim — APPROVED** [review]
## **Status: Approved** :tada: 

Hi @Zuhaib042 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-   Follow Githuh flow. :heavy_check_mark: 
-   Meaningful commit messages. :heavy_check_mark: 
-  HTML && CSS best practices. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Zuhaib042) [reply index.html:18]: @Qoosim your suggestion is highly appreciated. can you please elaborate further. what do you mean by hamburger icon being broken?

**Pull request merged**